module/network/RobotCommunication/src/RobotCommunication.cpp

```diff
@@ -211,6 +211,10 @@ namespace module::network {
                     msg->purpose = *purpose;
                 }
 
+
+                // Emit to the local network for debugging
+                emit(std::make_unique<RoboCup>(*msg));
+
```

Comment:
This may cause unwanted behaviour as we assume robocup messages come from other robots. You could use global config and change it to emit regularly. But would need to test that this doesn't break things too.